src/rest/study.ts

```diff
@@ -4,15 +4,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import { backendFetchJson } from '../utils/rest-api';
+import { backendFetchJson, Token } from '../utils/rest-api';
 
 const API_URL =
     '/api/' +
     (process.env.REACT_APP_USE_AUTHENTICATION === 'true'
-        ? process.env.REACT_APP_API_GATEWAY + '/study/v1'
-        : process.env.REACT_APP_SRV_STUDY_URI + '/v1');
+        ? `${process.env.REACT_APP_API_GATEWAY}/study/v1`
```

Comment:
Why is it better ?

Reply:
that is [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) syntax, that I found more easy to read.

Reply:
I though it was about performance, personnally I prefer the previous one but whatever